corehq/apps/data_vault/tests/test_utils.py

```diff
@@ -0,0 +1,8 @@
+from corehq.form_processor.steps import VaultPatternExtractor
```

Comment:
can we rename this file to just plain `tests/utils.py` since it does not actually contain any tests.

Reply:
https://github.com/dimagi/commcare-hq/pull/28161/commits/21ccc5908c69ab9573004d6bea7fe8d254f90cff